### PR TITLE
Add User guide

### PR DESCRIPTION
**Pull request by Zzmobie**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -57,25 +57,42 @@ Format: `help`
 === Adding a person: `add`
 
 Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Format: `add n/NAME [p/PHONE_NUMBER]... [e/EMAIL]... [a/ADDRESS]... [t/TAG]...`
 
 [TIP]
-A person can have any number of tags (including 0)
+A person can have any number of tags,phone numbers, emails or addresses (including 0)
```
Comment (donjar) [inline docs/UserGuide.adoc]: tags, phone numbers

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -57,25 +57,42 @@ Format: `help`
...
 
 Examples:
 
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/Bob p/91294232`
```
Comment (donjar) [inline docs/UserGuide.adoc:81]: Add an example with many phone numbers at once?

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -57,25 +57,42 @@ Format: `help`
...
+* MODE can be either "recent", "frequent" or "tag". If MODE is set to "tag", the field [tag/TAG] must also be included.
+* If the [m/MODE] field is empty, a list of all persons is shown
+* Using MODE = "recent" will return a list of the 5 most recently accessed entries
+* Using MODE = "frequent" will return a list of the 5 most frequently accessed entries
+* Using MODE = "tag" will return a list of all person that are tagged with TAG
+****
+
+Examples:
+
+* `list`
```
Comment (donjar) [inline docs/UserGuide.adoc:100]: Desc of this command (lists all persons)?

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -57,25 +57,42 @@ Format: `help`
...
 
-Shows a list of all persons in the address book. +
-Format: `list`
+Shows a list of persons in the address book. +
+Format: `list [m/MODE] [tag/TAG]`
+
+****
+* MODE can be either "recent", "frequent" or "tag". If MODE is set to "tag", the field [tag/TAG] must also be included.
+* If the [m/MODE] field is empty, a list of all persons is shown
+* Using MODE = "recent" will return a list of the 5 most recently accessed entries
```
Comment (donjar) [inline docs/UserGuide.adoc:93]: 5 looks quite arbitrary, would doing something like `list recent 3` be better?

File: docs/UserGuide.adoc (hunk 3 above)
Comment (Zzmobie) [reply docs/UserGuide.adoc:100]: Added this

File: docs/UserGuide.adoc (hunk 2 above)
Comment (Zzmobie) [reply docs/UserGuide.adoc:81]: Added an example with multiple email addresses

File: docs/UserGuide.adoc (hunk 4 above)
Comment (Zzmobie) [reply docs/UserGuide.adoc:93]: I'll leave it default at 5, it feels like for a socialite, they may not want so much unnecessary functionality. It's going to affect performance if we have to keep track of the positions of all the entries, so we should probably implement an upper bound to the maximum size of the list.

**Review by donjar — APPROVED** [review]
LGTM

**Pull request merged**